src/pybert/pybert.py

```diff
@@ -14,31 +13,23 @@
 
 Copyright (c) 2014 by David Banas; All rights reserved World wide.
 """
-from traits.etsconfig.api import ETSConfig
-# ETSConfig.toolkit = 'qt4'  # Yields unacceptably small font sizes in plot axis labels.
-# ETSConfig.toolkit = 'qt4.celiagg'   # Yields unacceptably small font sizes in plot axis labels.
-# ETSConfig.toolkit = 'qt.qpainter'  # Was causing crash on Mac.
-# ETSConfig.toolkit = 'qt.image'     # Program runs, but very small fonts in plot titles and axis labels.
-# ETSConfig.toolkit = 'wx'           # Crashes on launch.
```

Comment:
Would you restore these 6 lines, please?
I'm using them to experiment with, and provide feedback on, different GUI backend alternatives, which are under active current development.